File: app/src/main/java/com/spidroid/starry/activities/FollowingListActivity.java
package com.spidroid.starry.activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class FollowingListActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // إعادة استخدام نفس المنطق والـ layout الخاص بـ FollowersListActivity
        Intent intent = new Intent(this, FollowersListActivity.class);
        intent.putExtra(FollowersListActivity.EXTRA_USER_ID, getIntent().getStringExtra(FollowersListActivity.EXTRA_USER_ID));
        intent.putExtra(FollowersListActivity.EXTRA_LIST_TYPE, "following");
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/spidroid/starry/activities/FollowingListActivity.kt
package com.spidroid.starry.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class FollowingListActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // إعادة استخدام نفس المنطق والـ layout الخاص بـ FollowersListActivity
        val intent = Intent(this, FollowersListActivity::class.java)
        intent.putExtra(
            FollowersListActivity.Companion.EXTRA_USER_ID,
            getIntent().getStringExtra(FollowersListActivity.Companion.EXTRA_USER_ID)
        )
        intent.putExtra(FollowersListActivity.Companion.EXTRA_LIST_TYPE, "following")
        startActivity(intent)
        finish()
    }
}